spec/std/ini_spec.cr

```diff
@@ -1,5 +1,6 @@
 require "spec"
-require "ini"
+# require "ini"
+require "../../src/ini.cr"
```

Comment:
Please revert this change.

Reply:
How to run a spec specially for one filw? I always miss to revert this

Reply:
Yes but you don't have the updated library, here "ini". You need to rebuild crystal first, that is quite long on my old machine.

Reply:
Always use `./bin/crystal` instead of `crystal` and you'll use the stdlib in the repo instead of the installed stdlib.

Reply:
My bad, yes it was `bin/crystal` instead

Reply:
@j8r you don't need to rebuild crystal when you change the stdlib, you only need to do it when you change the compiler. Just use `bin/crystal` and you'll be fine.

Reply:
Good to know @RX14 , thanks. This avoids doing monkey patching but requires LLVM. I see there is a [doc about](https://github.com/crystal-lang/crystal/blob/master/CONTRIBUTING.md#the-standard-library), but a bit lost in all of this informations. I'll work on compliance command that will automates this (run `format` and run specs according modified files on git).